code-conversion/patterns/ALL_IN_ONE.md

```diff
@@ -75,9 +75,9 @@ The code conversion examples cover both Camunda 7 approaches to handle process v
 
```

Comment:
This file is auto generated from changes that were review in https://github.com/camunda/camunda-7-to-8-migration-tooling/pull/449